apps/barista/src/styles/global/_base.scss

```diff
@@ -83,6 +83,8 @@ code {
   font-family: 'Bitstream Vera', monospace;
 }
 
+$unordered-list-theme-color: $turquoise-500;
```

Comment:
Do we need this variable somewhere else? Otherwise I would remove this line and just use `$turquoise-500` below.